core/api/transport/network_address.hpp

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_API_TRANSPORT_NETWORK_ADDRESS_HPP
+#define KAGOME_CORE_API_TRANSPORT_NETWORK_ADDRESS_HPP
+
+#include <cstdint>
+#include <boost/asio/ip/address.hpp>
+
+namespace kagome::api {
+  using NetworkAddress = boost::asio::ip::address;
```

Comment:
better remove this using and use boost asio directly

Reply:
or even use `endpoint` instead of address + port

Reply:
removed, since it is no more required, replaced by endpoint some time ago.